src/main/avro/it/bitrock/dvs/model/FlightReceived.avdl

```diff
@@ -11,6 +11,8 @@ protocol FlightReceivedProtocol {
   record AirportInfo {
     string codeAirport;
     string nameAirport;
+    double latitudeAirport;
+    double longitudeAirport;
```

Comment:
We are in airport info why not simply `latitude` and `longitude`?
It could be a pain but also code and name should be renamed though it's out of scope.

Reply:
I agree but I tried to be consistent with the other name in the model

Reply:
yes please let's remove the Airport suffix